benchmark/src/bin_utils.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
if this util file is only used for bin, why don't we put it under `bin` folder?

Reply:
In addition to be used by binaries inside `benchmark` crate, these helper functions are used by the `testsuite` package. So I think it will be more convenient to put it into the `benchmark` library.